twigs/twigs.py

```diff
@@ -325,6 +326,35 @@ def main(args=None):
         parser_acr.add_argument('--no_code', action='store_true', help=argparse.SUPPRESS)
         parser_acr.add_argument('--sast', action='store_true', help=argparse.SUPPRESS)
 
+        
+        # Arguments required for Azure Container Registry container discovery 
+        parser_ecr = subparsers.add_parser ("ecr", help = "Discover AWS Container Registry (ECS) container images")
```

Comment:
Should it be "(ECR)" instead of "ECS" in the help message?

Reply:
Done.